src/index.html

```diff
@@ -9,6 +9,44 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header-box">
+    <a href="#" class="logo">
+      <img src="./images/logo.png" alt="logo">
+    </a>
+    <nav>
+      <ul class="nav-list">
+        <li class="nav-item">
+          <a class="links is-active" href="">Apple</a>
+        </li>
+        <li class="nav-item">
+          <a class="links" href="">Samsung</a>
+        </li>
+        <li class="nav-item">
+          <a class="links" href="">Смартфоны и телефоны</a>
```

Comment:
ты стилизуешь каждую конкретную ссылку, а не блок состоящий из ссылок, поэтому единственное число
```suggestion
          <a class="link" href="">Смартфоны и телефоны</a>
```